modules/kmm-configuring-kmmo.adoc

```diff
@@ -93,6 +93,9 @@ The recommended value is `true`.
 
 |`worker.setFirmwareClassPath`
 |Sets the kernel's firmware search path into the `/sys/module/firmware_class/parameters/path` file on the node. The recommended value is `/var/lib/firmware` if you need to set that value through the worker app. Otherwise, unset.
+
+|`job.gcDelay`
```

Comment:
@qbarrand Please verify this is the correct placement for this. Also, I think it needs to be added to the example file above the table.

Reply:
Could we follow the alphabetical order for those settings?
Agreed, this could be added to the example above:
```yaml
job:
  gcDelay: 1h
```

Reply:
Please check new placement in the example and the table.

Reply:
Looks good to me.